README.md

```diff
@@ -19,9 +19,9 @@ without interfering with day-to-day development using Bundler.
 Installation
 ------------
 
-In your Gemfile:
+In your package's .gemspec:
 
-    gem "appraisal"
+   s.add_development_dependency 'appraisal'
```

Comment:
Please could this be changed to double quotes, so it remains consistent with the rest of the readme?